frontend/__tests__/components/CardScreen.test.tsx

```diff
@@ -51,4 +51,12 @@ describe('CardScreen', () => {
 
     expect(screen.getByText(/failed to fetch/i)).toBeVisible()
   })
+
+  test('clicking on restaurant filter button opens filter options', async () => {
+    expect(screen.queryByText('Filters')).not.toBeInTheDocument()
+
+    await userEvent.click(screen.getByRole('button', { name: 'Filter Restaurants' }))
```

Comment:
I'm getting a `Unresolved variable or type userEvent` error. `userEvent` needs to be imported.  Were you able to run tests without it being imported?

Reply:
oh! totally! thanks for letting me know!